From 05d1d56cee46d41e320323ac1a23d90d741e3672 Mon Sep 17 00:00:00 2001
From: Oliwier Krawczyk <36317711+TheMysticle@users.noreply.github.com>
Date: Wed, 24 Jun 2026 12:17:34 +0200
Subject: [PATCH 4/7] Stub out call to jcodec MP4 post processing

---
 smali_classes7/tj/d.smali | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/smali_classes7/tj/d.smali b/smali_classes7/tj/d.smali
index 0f110378..a55cc30b 100644
--- a/smali_classes7/tj/d.smali
+++ b/smali_classes7/tj/d.smali
@@ -12,6 +12,8 @@
         }
     .end annotation
 
+    return-void
+
     :try_start_0
     new-instance v0, Loj/a;
 
-- 
2.54.0

